lib/shared/media_view.dart

```diff
@@ -278,6 +278,11 @@ class _MediaViewState extends State<MediaView> with SingleTickerProviderStateMix
           fit: widget.allowUnconstrainedImageHeight ? StackFit.loose : StackFit.expand,
           alignment: Alignment.bottomLeft,
           children: [
+            if (widget.postViewMedia.media.first.thumbnailUrl?.isNotEmpty != true)
```

Comment:
This could potentially clash with the warning icon for NSFW posts - we can add a quick check for that here:
```suggestion
            if (!blurNSFWPreviews && widget.postViewMedia.media.first.thumbnailUrl?.isNotEmpty != true)
```

Reply:
Good point! I committed your change!

Do you think the same could be true for the other change I made here below?

Reply:
Yeah, that might also be possible! I would also apply the checks when the image fails to load

Reply:
I did some testing and found that, regardless of the "blur" setting, if we don't have a thumbnail, we'll still display the warning icon, due to this line.

https://github.com/thunder-app/thunder/blob/768819324a39c2b7e00249b2f5df491e128c75be/lib/shared/media_view.dart#L292

So I think I will not display any other icon if the post is NSFW.

Let me know if this looks good to you now.